src/common/options/mon.yaml.in

```diff
@@ -764,6 +764,16 @@ options:
   services:
   - mon
   with_legacy: true
+- name: mds_beacon_mon_down_grace
+  type: float
+  level: advanced
+  desc: tolerance in seconds for missed MDS beacons to monitors
+  fmt_desc: The interval without beacons before Ceph declares an MDS laggy
+    when a monitor is down.
+  default: 60
+  services:
+  - mon
+  with_legacy: true
```

Comment:
if we don't read this setting on a critical path, and it is cached by a local variable. and we dispense with the `with_legacy` part?